spec/src/main/asciidoc/configsources.asciidoc

```diff
@@ -35,8 +35,6 @@ It can also be used to implement a drop-in configuration approach.
 Simply create a jar containing a `ConfigSource` with a higher ordinal and override configuration values in it.
 If the jar is present on the classpath then it will override configuration values from <<ConfigSource,ConfigSources>> with lower `ordinal` values.
 
-The ordinal for property file based <<ConfigSource,ConfigSources>> can be configured using the key `config_ordinal` inside the property file.
```

Comment:
I disagree.  The main problem with this line is that it implies you can add other property files, outside of `META-INF/microprofile-config.properties`.  We don't present support this outside of writing a custom property source.  Therefore we should not specify within 1.0 that you can do this, which is why I put it as part of the property file.

A change like what is being described means that we would need to ask the config source for the value of `config_ordinal` each time, which is potentially expensive.